json_serializable/lib/src/json_serializable_generator.dart

```diff
@@ -43,28 +43,41 @@ class JsonSerializableGenerator
       ].followedBy(_typeHelpers).followedBy(_coreHelpers);
 
   /// If `true`, wrappers are used to minimize the number of
-  /// [Map] and [List] instances created during serialization. This will
-  /// increase the code size, but it may improve runtime performance, especially
-  /// for large object graphs.
+  /// [Map] and [List] instances created during serialization.
+  ///
+  /// This will increase the code size, but it may improve runtime performance,
```

Comment:
These line lengths look inconsistent. Would `[Map]` have fit within 80 characters on line 45 or is this line going over 80 characters?

Reply:
I often wrap at the sentence start. It doesn't affect markdown rendering, but it doe make it easier to edit sentences without rewrapping the entire paragraph.

Reply:
neither of these lines are wrapped at a sentence start...